From 26dc9b5b7fef75bd05c0816236b6b4706bfae6b1 Mon Sep 17 00:00:00 2001
From: Stephan Linz <linz@li-pro.net>
Date: Tue, 31 Jul 2012 21:07:57 +0200
Subject: [PATCH] Prepare v2012.07-mbref

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 Makefile                             |    2 +-
 include/configs/microblaze-generic.h |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index a5a1db0..5ae574c 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@
 VERSION = 2012
 PATCHLEVEL = 07
 SUBLEVEL =
-EXTRAVERSION =
+EXTRAVERSION = -mbref
 ifneq "$(SUBLEVEL)" ""
 U_BOOT_VERSION = $(VERSION).$(PATCHLEVEL).$(SUBLEVEL)$(EXTRAVERSION)
 else
diff --git a/include/configs/microblaze-generic.h b/include/configs/microblaze-generic.h
index 0a349c3..432288f 100644
--- a/include/configs/microblaze-generic.h
+++ b/include/configs/microblaze-generic.h
@@ -389,7 +389,7 @@
 #endif
 
 /* Miscellaneous configurable options */
-#define	CONFIG_SYS_PROMPT	"U-Boot-mONStR> "
+#define	CONFIG_SYS_PROMPT	"U-Boot-mbref> "
 /* size of console buffer */
 #define	CONFIG_SYS_CBSIZE	512
  /* print buffer size */
-- 
1.7.0.4

