From de16f48906f3068c651458852c99e2634252cca0 Mon Sep 17 00:00:00 2001
Message-Id: <de16f48906f3068c651458852c99e2634252cca0.1284809613.git.linz@li-pro.net>
From: Stephan Linz <linz@li-pro.net>
Date: Sat, 18 Sep 2010 13:28:49 +0200
Subject: [PATCH 1/1] microblaze: generic: move ENV onto top of flash

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 include/configs/microblaze-generic.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/configs/microblaze-generic.h b/include/configs/microblaze-generic.h
index 9a52af5..dc5f3c9 100644
--- a/include/configs/microblaze-generic.h
+++ b/include/configs/microblaze-generic.h
@@ -169,7 +169,7 @@
 	#else	/* !RAMENV */
 		#define	CONFIG_ENV_IS_IN_FLASH	1
 		#define	CONFIG_ENV_SECT_SIZE	0x20000	/* 128K(one sector) for env */
-		#define	CONFIG_ENV_ADDR		(CONFIG_SYS_FLASH_BASE + (2 * CONFIG_ENV_SECT_SIZE))
+		#define CONFIG_ENV_ADDR		(CONFIG_SYS_FLASH_BASE + CONFIG_SYS_FLASH_SIZE - CONFIG_ENV_SECT_SIZE)
 		#define	CONFIG_ENV_SIZE		0x20000
 	#endif /* !RAMBOOT */
 #else /* !FLASH */
-- 
1.6.0.4

