From 41a6616ac3cb68e58f7439c89a01636f907ee8ba Mon Sep 17 00:00:00 2001
From: Stephan Linz <linz@li-pro.net>
Date: Fri, 28 Jun 2013 17:52:58 +0200
Subject: [PATCH] Prepare v2013.04-mbref

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 Makefile                             | 2 +-
 include/configs/microblaze-generic.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index e51cc52..8ebcc53 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@
 VERSION = 2013
 PATCHLEVEL = 04
 SUBLEVEL =
-EXTRAVERSION =
+EXTRAVERSION = -mbref
 ifneq "$(SUBLEVEL)" ""
 U_BOOT_VERSION = $(VERSION).$(PATCHLEVEL).$(SUBLEVEL)$(EXTRAVERSION)
 else
diff --git a/include/configs/microblaze-generic.h b/include/configs/microblaze-generic.h
index 3d4f2f4..68a3baa 100644
--- a/include/configs/microblaze-generic.h
+++ b/include/configs/microblaze-generic.h
@@ -385,7 +385,7 @@
 #endif
 
 /* Miscellaneous configurable options */
-#define	CONFIG_SYS_PROMPT	"U-Boot-mONStR> "
+#define	CONFIG_SYS_PROMPT	"U-Boot-mbref> "
 /* size of console buffer */
 #define	CONFIG_SYS_CBSIZE	512
  /* print buffer size */
-- 
1.8.3.4

