From 9adac4976a326c0ed24b57047244d39dfc2cac3f Mon Sep 17 00:00:00 2001
From: Stephan Linz <linz@li-pro.net>
Date: Sat, 7 Dec 2013 12:49:13 +0100
Subject: [PATCH] net: xilinx: ll_temac: access to NATIONAL DP83865 config
 option

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 drivers/net/ethernet/xilinx/Kconfig | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/drivers/net/ethernet/xilinx/Kconfig b/drivers/net/ethernet/xilinx/Kconfig
index 0bbbe5e..8c9126e 100644
--- a/drivers/net/ethernet/xilinx/Kconfig
+++ b/drivers/net/ethernet/xilinx/Kconfig
@@ -54,6 +54,14 @@ choice
 	depends on XILINX_LLTEMAC
 	default XILINX_LLTEMAC_MARVELL_88E1111_GMII
 
+config XILINX_LLTEMAC_NATIONAL_DP83865_GMII
+	bool "NATIONAL DP83865 using GMII"
+   help
+	  This phy is used by some Xilinx boards.  This option includes
+	  code for enabling GMII over copper, and for setting the correct
+	  speed based on whatever the phy is able to autonegotiate.  This is
+	  usually the best option to use on SP3ADSP1800 boards.
+
 config XILINX_LLTEMAC_MARVELL_88E1111_RGMII
 	bool "MARVELL 88E1111 using RGMII"
    help
-- 
1.8.3.4

