From dec686cfc2fd2fc2f376e3c7d46c2f1007951902 Mon Sep 17 00:00:00 2001
From: Michal Simek <monstr@monstr.eu>
Date: Fri, 27 May 2011 15:27:42 +0200
Subject: [PATCH] axi_ethernet: Kick off DMA transfer asap

Signed-off-by: Michal Simek <monstr@monstr.eu>
---
 drivers/net/xilinx_axienet_main.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/xilinx_axienet_main.c b/drivers/net/xilinx_axienet_main.c
index 1cff0c1..7a98a08 100755
--- a/drivers/net/xilinx_axienet_main.c
+++ b/drivers/net/xilinx_axienet_main.c
@@ -814,13 +814,13 @@ static int axienet_start_xmit(struct sk_buff *skb, struct net_device *ndev)
 	cur_p->cntrl |= XAXIDMA_BD_CTRL_TXEOF_MASK;
 
 	tail_p = lp->tx_bd_p + sizeof(*lp->tx_bd_v) * lp->tx_bd_tail;
+	/* Start the transfer */
+	axienet_dma_out32(lp, XAXIDMA_TX_TDESC_OFFSET, tail_p);
+
 	lp->tx_bd_tail++;
 	if (lp->tx_bd_tail >= TX_BD_NUM)
 		lp->tx_bd_tail = 0;
 
-	/* Start the transfer */
-	axienet_dma_out32(lp, XAXIDMA_TX_TDESC_OFFSET, tail_p);
-
 	return NETDEV_TX_OK;
 }
 
-- 
1.7.0.4

