From b08897ebee41c1976c9fa016c40ff5bb43077a2e Mon Sep 17 00:00:00 2001
From: Michal Simek <monstr@monstr.eu>
Date: Mon, 30 May 2011 14:09:37 +0200
Subject: [PATCH] axi_ethernet: Move frag loading when necessary

Signed-off-by: Michal Simek <monstr@monstr.eu>
---
 drivers/net/xilinx_axienet_main.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/drivers/net/xilinx_axienet_main.c b/drivers/net/xilinx_axienet_main.c
index 7a98a08..d1eacf1 100755
--- a/drivers/net/xilinx_axienet_main.c
+++ b/drivers/net/xilinx_axienet_main.c
@@ -758,7 +758,6 @@ static int axienet_start_xmit(struct sk_buff *skb, struct net_device *ndev)
 	u32 csum_index_off;
 
 	num_frag = skb_shinfo(skb)->nr_frags;
-	frag = &skb_shinfo(skb)->frags[0];
 	start_p = lp->tx_bd_p + sizeof(*lp->tx_bd_v) * lp->tx_bd_tail;
 	cur_p = &lp->tx_bd_v[lp->tx_bd_tail];
 
@@ -803,13 +802,13 @@ static int axienet_start_xmit(struct sk_buff *skb, struct net_device *ndev)
 
 		cur_p = &lp->tx_bd_v[lp->tx_bd_tail];
 		cur_p->cntrl = 0;
+		frag = &skb_shinfo(skb)->frags[ii];
 		cur_p->phys = dma_map_single(ndev->dev.parent,
 			(void *)page_address(frag->page) + frag->page_offset,
 			frag->size,
 			DMA_TO_DEVICE);
 		cur_p->cntrl = ((cur_p->cntrl & (~XAXIDMA_BD_CTRL_LENGTH_MASK))
 								| frag->size);
-		frag++;
 	}
 	cur_p->cntrl |= XAXIDMA_BD_CTRL_TXEOF_MASK;
 
-- 
1.7.0.4

